lib/build_runner.zig

```diff
@@ -192,6 +192,12 @@ pub fn main() !void {
                     std.debug.print("Expected argument after {s}\n\n", .{arg});
                     usageAndErr(builder, false, stderr_stream);
                 } };
+            } else if (mem.eql(u8, arg, "--seed")) {
+                const next_arg = nextArg(args, &arg_idx) orelse {
+                    std.debug.print("Expected a string after {s}\n\n", .{arg});
+                    usageAndErr(builder, false, stderr_stream);
+                };
+                builder.seed = next_arg;
```

Comment:
should use a local variable instead of storing extra state into Build

Reply:
Right I didn't realize this was the same scope but it's indeed better.